app/experimenter/experiments/constants/nimbus.py

```diff
@@ -92,6 +93,64 @@ class NimbusTargetingConfig:
 )
 
 
+class Channel(models.TextChoices):
+    NO_CHANNEL = ""
+    UNBRANDED = "default"
+    NIGHTLY = "nightly"
+    BETA = "beta"
+    RELEASE = "release"
+
+
+@dataclass
+class ApplicationConfig:
```

Comment:
Ah, yup, this looks like a nice inversion of the separate little constants into better related records